src/pseudopeople/interface.py

```diff
@@ -492,9 +501,10 @@ def generate_taxes_1040(
         to a configuration YAML file, a configuration dictionary,
         or the sentinel value `pseudopeople.NO_NOISE`, which will generate a
         dataset without any configurable noise.
-    :param year: The tax year (format YYYY) to include in the dataset. Will raise
-        a ValueError if there is no data with this year. If None is provided,
-        data from all years are included in the dataset.
+    :param year: The tax year for which to generate records (format YYYY, e.g., 2036);
+        the simulated dataset will contain the 1040 tax forms filed by individual simulants for the specified year. Will raise
```

Comment:
```suggestion
        the simulated dataset will contain the 1040 tax forms filed by simulants for the specified year. Will raise
```

Or is this word doing some work I don't understand? I just thought it might be confusing since multiple simulants can appear on a 1040 in different roles.

Reply:
At first I just said "individuals" since the name of the 1040 is "Individual Income Tax Return", then I thought I should say "simulants", so I said "individual simulants." I guess we should use one or the other.